src/cli.rs

```diff
@@ -143,12 +166,59 @@ pub struct Config {
         short = 'T',
         arg_enum,
     )]
+    #[merge(strategy = overwrite_vector)]
     pub r#type: Vec<LanguageType>,
+    /// Specify a custom path to a config file.
+    /// Default config is located at ${HOME}/.config/onefetch/config.conf.
```

Comment:
Just an FYI that this is only the default path on Linux if you're using `dirs`, AFAIK.